src/Microsoft.FeatureManagement/FeatureFilters/TimeWindowFilter.cs

```diff
@@ -43,7 +43,9 @@ public object BindParameters(IConfiguration filterParameters)
         /// <returns>True if the feature is enabled, false otherwise.</returns>
         public Task<bool> EvaluateAsync(FeatureFilterEvaluationContext context)
         {
-            TimeWindowFilterSettings settings = (TimeWindowFilterSettings)context.Settings;
+            //
+            // Assume settings was bound from feature manager, if not, bind from parameters
```

Comment:
> bound from feature manager

I don't like how the comment mentions an expected caller. Public method can be called in any way.

I'd suggest
> Check if prebound settings available, otherwise bind from parameters.